Full-Stack-Collective/clerkwise
# Jf set up husky

## Description

Adds consistent styling to the project.

## Related Issue

Closes https://github.com/Full-Stack-Collective/clerkwise/issues/27

## Acceptance Criteria

- [x] Husky and lint-staged are set up

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
| ✓   | :sparkles: New feature     |

## Testing Steps / QA Criteria
1. Switch to branch
2. Run `npm ci`
3. Try changing a file
4. Commit the file
5. Husky should run
